### PR TITLE
Image input components ready to merge

### PR DESCRIPTION
**Pull request by b0llu**
I am done with my image and input components do give feedback
image
image
image
image
image

### CODE REVIEW

File: Components/Alert/Alert.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.success-alert {
+  background-color: var(--success-color);
+  color: #000000;
+}
+
+.warning-alert {
+  background-color: var(--warning-color);
+  color: #000000;
```
Comment (anik31) [inline Components/Alert/Alert.css]: You can use color variables so that you can change them at one place if you want to

Comment (b0llu) [reply Components/Alert/Alert.css]: thanks for that I just changed it!

**Comment by vashnavichauhan1825** [comment]
looks good to me great work !

**Comment by hsnice16** [comment]
Nice work man. Great work on UI.

**Pull request merged**